modules/hal_sensors/sensor_differentialpressure/src/sensor_differentialpressure.cpp

```diff
@@ -52,11 +53,35 @@ void DifferentialPressureSensor::BeginSDP810() {
 }
 
 void DifferentialPressureSensor::ReadSdp810() {
-  i2c_handle_->ReadBytes(sensor_buffer_, kSdp810BufferSize);
+  #define pi 3.141592653
+  #define m3tocm3_factor 1000000
+  #define air_weight 1.29
+  sampleTime = 10;
+  air_drag = 0.61;
+  hose_orifice_ratio = 0.5;
+  orifice_diameter = 0.003;
+  correction_factor = 1.525;
+  orifice_surface = pi / 4.0 * pow(orifice_diameter, 2);
```

Comment:
These seem to be constants.. Please use a macro or constexpr to evaluate the constant value at compile time. This saves resources.

Reply:
And what is the magic number 4.0